server.js

```diff
@@ -1,7 +1,41 @@
 const express = require("express");
+const axios = require("axios");
 
-const app = express();
+require("dotenv").config();
```

Comment:
I would move this to the top of this file

> As early as possible in your application, require and configure dotenv.

https://www.npmjs.com/package/dotenv#usage